alerts/elasticsearch-gke/README.md

```diff
@@ -0,0 +1,36 @@
+# Elasticsearch Alerts for GKE
```

Comment:
As a general question, what are your thoughts about adding potential resolution steps in the documentation? Are there any common best practices that if the failure occurs, what are common remediation steps?

Reply:
Generally, remediation steps fall outside of this scope of work. More generally, we're more hesitant to recommend 'common' remediation steps due to the risk/variability of customer environments.